apps/backend/src/constants.ts

```diff
@@ -20,8 +21,9 @@ export const OAUTH_APP_MAPPING: Record<string, OAuthMapping> = {
   },
   "github": {
     client_side_fetching: false,
+    can_import: false,
     token_url: "https://github.com/login/oauth/access_token",
-    redirect_uri: "http://connections.cursive.team/oauth/exchange_token&approval_prompt=force&scope=read",
+    redirect_uri: "http://localhost:3000/oauth/exchange_token&approval_prompt=force&scope=read",
```

Comment:
Looks like this was accidentally pushed, removing and updating to https while I'm at it.